src/lambda/commands/createNewSamApp.ts

```diff
@@ -53,7 +53,7 @@ import { execSync } from 'child_process'
 
 type CreateReason = 'unknown' | 'userCancelled' | 'fileNotFound' | 'complete' | 'error'
 
-export const SAM_INIT_TEMPLATE_FILE: string = 'template.yaml'
+export const SAM_INIT_TEMPLATE_FILES: string[] = ['template.yaml', 'template.yml']
```

Comment:
The other consideration I would make would be to instead have a mapping between SAM starter app type and SAM template type in case SAM starts including more template names.